### PR TITLE
Always increment CE actual response count where applicable 

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Motivation and Context
The CE actual response count should be incremented regardless of whether the case is receipted or not.

# What has changed
* Always look up and run the receipting rule
* Check receipted status in receipting rules

# How to test?
Build and run linked AT's

# Links
https://trello.com/c/bhDgGx3H/1035-defect-ceactualresponse-count-not-increasing-for-ce-case
https://github.com/ONSdigital/census-rm-acceptance-tests/pull/275

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[Defect: ceActualresponse count not increasing for CE case](https://trello.com/c/bhDgGx3H/1035-defect-ceactualresponse-count-not-increasing-for-ce-case)

**Comment by AdamHawtin** [comment]
[Defect: ceActualresponse count not increasing for CE case](https://trello.com/c/bhDgGx3H/1035-defect-ceactualresponse-count-not-increasing-for-ce-case)

File: src/test/java/uk/gov/ons/census/casesvc/service/CaseReceiptServiceTableTest.java [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+    try {
+      underTest.receiptCase(uacQidLink, getReceiptEventDTO());
+    } catch (RuntimeException ex) {
+      if (expectation.isRunTimeExceptionExpected()) {
+        verifyNoInteractions(caseService);
+        verifyNoInteractions(caseRepository);
+        return;
+      }
+
+      fail("Unexepcted exception" + ex.getMessage() + " for " + key.toString());
```
Comment (ryangrundy7) [inline src/test/java/uk/gov/ons/census/casesvc/service/CaseReceiptServiceTableTest.java:125]: Spelling mistake

File: src/test/java/uk/gov/ons/census/casesvc/service/CaseReceiptServiceTest.java [hunk 2]
```diff
@@ -239,9 +119,9 @@ public void testEqReceiptForQidMarkedBlankQuestionnaireReceiptsCase() {
   private String getQid(String qidType) {
```
Comment (ryangrundy7) [inline src/test/java/uk/gov/ons/census/casesvc/service/CaseReceiptServiceTest.java:119]: This isn't used now

**Pull request merged**